compose/.apps/rustdesk/rustdesk.x86_64.yml

```
@@ -0,0 +1,3 @@
+services:
+  rustdesk:
+    image: rustdesk/rustdesk-server-s6:latest
```

```suggestion
    image: rustdesk/rustdesk-server-s6
```
`latest` is implied, and thus not needed.